content/influxdb/cloud-dedicated/reference/cli/influxctl/query.md

```diff
@@ -0,0 +1,235 @@
+---
+title: influxctl query
+description: >
+  The `influxctl query` command queries data from InfluxDB Cloud Dedicated
+  using SQL and prints results as a table or JSON.
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl
+weight: 201
+metadata: [influxctl 2.4.0+]
+related:
+  - /influxdb/cloud-dedicated/reference/sql/
+  - /influxdb/cloud-dedicated/query-data/
+---
+
+The `influxctl query` command queries data from {{< product-name >}} using SQL
+and prints results as a table or JSON.
+
+Provide the SQL query in one of the following ways:
+
+- a string on the command line
+- a path to a file that contains the query
+- as a single dash (`-`) to read the query from stdin
+
+{{% note %}}
+#### Important to note
+
+- This command supports only one query per execution.
+- This command supports only SQL queries; not InfluxQL.
+- This command is not meant to be a full, feature-rich query tool.
+  It's meant for debug, triage, and basic data exploration.
+{{% /note %}}
+
+### InfluxDB connection configuration
+
+Your {{< product-name omit=" Clustered" >}} cluster host and port should be
```

Comment:
> should be

They are or I need to add them?

Reply:
They're part of the sample config. They aren't necessarily required with Dedicated, unless you want to use these `write` and `query` commands.